src/xdocs/config_misc.xml

```diff
@@ -1320,6 +1320,118 @@ void foo(String aFooString,
       </subsection>
     </section>
 
+    <section name="OrderedProperties">
+      <subsection name="Description" id="OrderedProperties_Description">
+          <p>Since Checkstyle 8.21</p>
+          <p>
+              Checks if property keys (left from equal sign) are sorted.
+          </p>
+          <p>
+              Rationale: Unsorted property makes merges more easy. While there are no errors in
+              runtime.
+              Important is the binary order of keys where 'only.text' before 'only.text.sub'.
+          </p>
+          <p>
+              Known limitation: The key should not contain a newline.
+              The string compare will work, but not the line number reporting.
+          </p>
+      </subsection>
+
+      <subsection name="Properties" id="OrderedProperties_Properties">
+          <table>
+              <tr>
+                  <th>name</th>
+                  <th>description</th>
+                  <th>type</th>
+                  <th>default value</th>
+                  <th>since</th>
+              </tr>
+              <tr>
+                  <td>fileExtensions</td>
+                  <td>file type extension of the files to check.</td>
+                  <td><a href="property_types.html#stringSet">String Set</a></td>
+                  <td>.properties</td>
+                  <td>8.21</td>
+              </tr>
+          </table>
+      </subsection>
+
+      <subsection name="Examples" id="OrderedProperties_Examples">
+          <p>
+              To configure the check:
+          </p>
+          <source>
+              &lt;module name=&quot;OrderedProperties&quot; /&gt;
+          </source>
+          <p>
+              Example properties file:
+          </p>
+          <source>
+              @=64
+              A=65
+              a=97
+              key=107 than nothing
+              key.sub=k is 107 and dot is 46
+          </source>
```

Comment:
please extend example to show case with violation as `key.p=value //violation`

Reply:
done